packages/url/src/url.ts

```diff
@@ -47,3 +47,7 @@ export function withProtocol(
     ? url
     : `${protocol}://${url}`;
 }
+
+export function withoutProtocol(url: string): string {
+  return url.replace(/^(http:\/\/|https:\/\/|tel:|mailto:)/, '');
```

Comment:
Would be nice to have some tests for this.

Reply:
Added in 913d5718cda313916412358a4e13098b4e23f722.